### PR TITLE
Point out you can use linuxbrew in README

### PR DESCRIPTION
**Pull request by fabiosantoscode**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ Reading package lists... Done
 
 ## Installation
 
-On OS X, you can install *The Fuck* via [Homebrew][homebrew]:
+On OS X or linux systems with linuxbrew, you can install *The Fuck* via [Homebrew][homebrew]:
```
Comment (scorphus) [inline README.md]: I'd rather like it to read something like:

```
On OS X, you can install *The Fuck* via [Homebrew][homebrew] (or via [Linuxbrew][linuxbrew] on Linux):
```

Comment (fabiosantoscode) [reply README.md]: Done

**Comment by scorphus** [comment]
What's your opinion, @josephfrazier?

**Comment by fabiosantoscode** [comment]
Done, changed brew's URL to https too

**Comment by scorphus** [comment]
Thanks, @fabiosantoscode!

**Pull request merged**